nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -1,20 +1,26 @@
 #!/usr/bin/env python3
+import queue
+from enum import Enum
 
 import rospy
 import actionlib
-import queue
-
 from actionlib_msgs.msg import GoalStatus
 from std_msgs.msg import String
 from nightingale_msgs.msg import MissionPlanAction
 from nightingale_dispatcher.navigate_task import NavigateTask
 from nightingale_dispatcher.move_arm_task import MoveArmTask
 from nightingale_dispatcher.send_interface_request_task import SendInterfaceRequestTask
-from nightingale_dispatcher.task import Task
+from nightingale_dispatcher.task import Task, TaskCodes
 from nightingale_ros_bridge.bridge_interface_config import BridgeConfig, RobotStatus
 
 
-class MissionPlanner:
+# enum for phase status
+class PhaseStatus(Enum):
+    PHASE_COMPLETE = 1
+    PHASE_FAIL = 0
+
+
+class MissionPlanner(PhaseStatus):
```

Comment:
i think this inheritance is not necessary. MissionPlanner shouldnt inherit from PhaseStatus you should be able to just delete it since it looks like ur using the enum correctly (ie. PhaseStatus.PHASE_COMPLETE)

Reply:
actually this might break things. I ran the mission planner but didn't get an error about inheriting the enum which is strange. I'll be removing the inheritance thanks